Libplanet.RocksDBStore/RocksDBStore.cs

```diff
@@ -22,6 +22,7 @@ namespace Libplanet.RocksDBStore
     /// <seealso cref="IStore"/>
     public class RocksDBStore : BaseStore
     {
+        #pragma warning disable MEN002 // Line is too long
```

Comment:
This warning shouldn't be disabled.  How about defining a simple method to log an unexpected exception?  For example:

```csharp
private string LogUnexpectedException(string methodName, Exception e)
{
    string msg = $"An unexpected exception occurred on {methodName}: {{Message}}";
    _logger.Error(e, msg, e.Message);
}
```